std/examples/ws/server.ts

```diff
@@ -0,0 +1,57 @@
+// Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
+import { serve } from "../http/server.ts";
```

Comment:
I think ws examples can still live in `/std/ws`.

I like having an examples directory for quickly showing people up a few examples, but the intention isn't to have every example program here.